src/main/java/org/apache/parquet/hadoop/BdecParquetWriter.java

```diff
@@ -45,6 +45,9 @@ public class BdecParquetWriter implements AutoCloseable {
   private final Optional<Integer> maxRowGroups;
   private long rowsWritten = 0;
 
```

Comment:
nit: add the final ones above the non-final ones!

Reply:
Moved to top